appveyor.yml

```diff
@@ -8,6 +8,16 @@ init:
 # Uncomment the following line to show RDP info at beginning of job
 #  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 clone_folder: c:\projects\mixxx
+skip_commits:
+  files:
+    - doc/
+    - res/
```

Comment:
We have a unit test that tests all the Javascript and XML in res/controllers parses without error and the schema.xml file is covered by a variety of tests for the database. 

Maybe stick res/skins, res/images, res/translations in here explicitly?